language/move-prover/boogie-backend/src/bytecode_translator.rs

```diff
@@ -528,6 +529,28 @@ impl<'env> ModuleTranslator<'env> {
             Label(_, label) => {
                 self.writer.unindent();
                 emitln!(self.writer, "L{}:", label.as_usize());
+                let annotated_func_target = self.targets.get_annotated_target(fun_target.func_env);
+                let loop_annotation = annotated_func_target
+                    .get_annotations()
+                    .get::<LoopAnnotation>()
+                    .expect("loop annotation");
+                if loop_annotation.loop_targets.contains_key(label) {
```

Comment:
Add a TODO here along "revisit whether can express what is needed here on bytecode level"?